.github/workflows/push.yaml

```diff
@@ -194,12 +193,36 @@ jobs:
           features: ${{ toJson(matrix.features) }}
           action: ${{ matrix.action }}
 
+  check-gitbutler-core:
```

Comment:
I think you may also wanna delete `check-gitbutler` on line `104` since its for the same target

Reply:
Great catch, thanks! Didn't realize it was for core.